principles.md

```diff
@@ -0,0 +1,33 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**I. Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms. This includes being coercion-resistant.
+
+**II. Censorship resistance**
+We enable free flow of information. No content is under surveillance. We abide by the cryptoeconomic design principle of censorship resistance. Even stronger, Status is an agnostic platform for information.
+
+**III. Security**
+We don't compromise on security when building features. We use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+**IV. Privacy**
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as being a pseudo-anonymous platform. Additionally, we strive to provide the right of total anonymity.
+
+**V. Transparency**
+We strive for complete openness and symmetry of information, and have no border between our core contributors and our community. We are frank about our shortcomings, especially when making short-term tradeoffs in service of our long-term goals.
+
+**VI. Openness**
+The software we create is a public good. It is made available via a free and open source license, for anyone to share, modify and benefit from. We believe in permission-less participation.
+
+**VII. Decentralization**
+We minimize centralization across both the software and the organization itself. In other words, we maximize the number of physical computers composing the network, and maximize the number of individuals who have control over the system(s) we are building.
+
+**VIII. Inclusivity**
+We believe in fair and widespread access to our software, with an emphasis on ease-of-use. This also extends to social inclusivity, permissionless participation, interoperability, and investing in educational efforts.
```

Comment:
@3esmit "interoperability" seems is related to "standardization", but standardized thing is not about survival or quick mutation as response to external influences. Such "interoperability" is good thing, but after the win is taken.